softlayer/common/fakes/fake_vm.go

```diff
@@ -141,6 +141,12 @@ type FakeVM struct {
 	updateAgentEnvReturns struct {
 		result1 error
 	}
+	DeleteAgentEnvStub        func() error
+	deleteAgentEnvMutex       sync.RWMutex
+	deleteAgentEnvArgsForCall []struct{}
+	deleteAgentEnvReturns     struct {
+		result1 error
```

Comment:
What's this `result1` business? Can we use a better name? Is this auto-generated?